### ISSUE TITLE
Timestamps in logs

### ISSUE THREAD

**Issue by zenhack** [sandstorm-io/sandstorm#3326]
It would be useful for troubleshooting if log messages had timestamps in them. This came up in IRC discussion around #3324

**Comment by ocdtrekkie** [comment]
I am positive I've had this opinion a few times before.

**Comment by ocdtrekkie** [comment]
This is actually a dupe of #1486, so I am going to close it. But I still think it's really important. FWIW, the original issue has an interesting "stopgap solution" idea.

**Comment by zenhack** [comment]
fwiw, that issue only talks about grain logs, whereas this has come up in the context of sandstorm.log several times. But maybe we should use that issue to track both anyway.

**Comment by ocdtrekkie** [comment]
Good catch. I think it makes sense to have separate issues in case they are resolved separately.
